src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -111,8 +111,12 @@ public SmartClinicalScopesMiddleware(RequestDelegate next, ILogger<SmartClinical
                                 }
 
                                 fhirRequestContext.AccessControlContext.AllowedResourceActions.Add(new ScopeRestriction(resource, permittedDataActions, id));
-
-                                _logger.LogInformation("Resource and permitted data actions {Resource} {PermittedDataActions} ", resource, permittedDataActions);
+                                scopeRestrictions.Append(" ( ");
+                                scopeRestrictions.Append(resource);
+                                scopeRestrictions.Append(" - ");
+                                scopeRestrictions.Append(permittedDataActions);
+                                scopeRestrictions.Append(" ) ");
+                                scopeRestrictions.Append(" \n ");
```

Comment:
Wondering how new line character works in geneva, does it create new rows with same values for column.
What about having this logged in the same line and we can probably use other separator(e.g. comma)?

Reply:
Done.